lib/hanami/action/application_action.rb

```diff
@@ -49,15 +62,8 @@ def resolve_view_context
         end
       end
 
-      def resolve_paired_view(action_class)
-        view_identifiers = application.config.actions.view_name_inferrer.(
-          action_name: action_class.name,
-          provider: provider
-        )
-
-        view_identifiers.detect { |identifier|
-          break provider[identifier] if provider.key?(identifier)
-        }
+      def resolve_routes
+        application[:routes_helper] if application.key?(:routes_helper)
```

Comment:
Isn't it defensive checking that `:router_helper` exists? Shouldn't it always exist as we're registering it from `hanami/hanami`?

Reply:
Good pick up :) At the moment, it's over-defensive, I agree! However, I would like at some point to move all the router integration opt-in/out like the rest of the system, which means we could very well have applications without a routes_helper. I'd much rather have this do the right thing from the beginning rather than have to be changed later. Does that make sense?